tests/kickstarts/test_suite.cfg

```diff
@@ -68,31 +79,55 @@ clearpart --linux --initlabel
 # Create primary system partitions (required for installs)
 part /boot --fstype=xfs --size=512
 part pv.01 --grow --size=1
-volgroup VolGroup pv.01
-logvol / --fstype=xfs --name=LogVol01 --vgname=VolGroup --size=3072
-logvol /home --fstype=xfs --name=LogVol02 --vgname=VolGroup  --size=1024
-logvol /tmp --fstype=xfs --name=LogVol03 --vgname=VolGroup --size=1024
-logvol /var --fstype=xfs --name=LogVol04 --vgname=VolGroup --size=2048
-logvol /var/log --fstype=xfs --name=LogVol05 --vgname=VolGroup --size=512
-logvol /var/log/audit --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=512
-logvol swap --name=lv_swap --vgname=VolGroup --size=1024
+
+# Create a Logical Volume Management (LVM) group (optional)
+volgroup VolGroup --pesize=4096 pv.01
+
+# Create particular logical volumes (optional)
+logvol / --fstype=xfs --name=root --vgname=VolGroup --size=12288 --grow
+# CCE-26557-9: Ensure /home Located On Separate Partition
+logvol /home --fstype=xfs --name=home --vgname=VolGroup --size=1024 --fsoptions="nodev"
+# CCE-26435-8: Ensure /tmp Located On Separate Partition
+logvol /tmp --fstype=xfs --name=tmp --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# CCE-26639-5: Ensure /var Located On Separate Partition
+logvol /var --fstype=xfs --name=var --vgname=VolGroup --size=2048 --fsoptions="nodev"
+# CCE-26215-4: Ensure /var/log Located On Separate Partition
+logvol /var/log --fstype=xfs --name=log --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# CCE-26436-6: Ensure /var/log/audit Located On Separate Partition
+logvol /var/log/audit --fstype=xfs --name=audit --vgname=VolGroup --size=512 --fsoptions="nodev,nosuid,noexec"
+logvol swap --name=swap --vgname=VolGroup --size=2016
 
 # Packages selection (%packages section is required)
 %packages
 
-# requirements of SSG Test Suite
+# Install selected additional packages (required by PCI-DSS profile)
+# CCE-27024-9: Install AIDE
+aide
+
+# Install libreswan package
+libreswan
```

Comment:
Why install these packages by default?
A test scenario should assume the bare minimum is installed in the image.
By having some packages already installed is confusing.

Reply:
Agree